cf_remote/ssh.py

```diff
@@ -107,39 +131,41 @@ def scp(file, remote, connection=None, rename=None):
 
 def ssh_cmd(connection, cmd, errors=False):
     assert connection
-    try:
-        log.debug(f"Running over SSH: '{cmd}'")
-        result = connection.run(cmd, hide=True)
+
+    log.debug(f"Running over SSH: '{cmd}'")
+    result = connection.run(cmd, hide=True)
+    if result.retcode == 0:
         output = result.stdout.replace("\r\n", "\n").strip("\n")
         log.debug(f"'{cmd}' -> '{output}'")
         return output
-    except UnexpectedExit as e:
-        msg = f"Non-sudo command unexpectedly exited: '{cmd}'"
+    else:
```

Comment:
else clause not needed

Reply:
Again, it means fewer lines changed.

Reply:
Yea, but I agree with @larsewi, reducing the nesting would be better.